prody/proteins/interactions.py

```diff
@@ -3751,23 +3773,96 @@ def calcLigandInteractionsTrajectory(self, atoms, trajectory=None, filename=None
             
         return interactions_all
 
-
+    
     def getLigandInteractions(self, **kwargs):
-        """Return the list of protein-ligand interactions."""
-
-        return self._interactions_traj
-
+        """Return the list of protein-ligand interactions.
+                
+        :arg filters: selection string of ligand with chain ID or interaction type
+                     e.g. 'SBs' (HBs, SBs, HPh, PiStack, PiCat, HPh, watBridge)
+        :type filters: str    
+        
+        :arg include_frames: used with filters, it will leave selected keyword in orginal 
+                    lists, if False it will collect selected interactions in one list,
+                    Use True to assign new selection using setLigandInteractions.
+                    by default True
+        :type include_frames: True or False            
+        """
+        
+        filters = kwargs.pop('filters', None)
+        include_frames = kwargs.pop('include_frames', True)
+        filtered_lists = []
+                
+        if filters != None:
+            if include_frames == False:
+                interactions = [element for group in self._interactions_traj for element in group]
```

Comment:
You can combine these two lines as `interactions = [element for group in self._interactions_traj for element in group if filters in element]`

Reply:
Ok, I replaced it. But I believe it should be:
**filtered_lists** = [element for group in self._interactions_traj for element in group 
                                if filters in element]

Reply:
I don’t see a difference between what you said and what I said

Reply:
The name of the list cannot be interactions but filtered_lists. The rest is the same.

Reply:
oh ok, of course